docs/3.2. Discovery - Peer to Peer Operation.md

```diff
@@ -15,6 +15,41 @@ The following pre-requisites are optional aspects of the [Node API](../APIs/Node
 
 Both of the above are properties of the Node API and require no additional implementation steps.
 
+## DNS-SD Advertisement
+
+Node APIs MAY produce an mDNS advertisement of the type \_nmos-node.\_tcp in order to support peer to peer operation. This SHOULD NOT be advertised when a Registration API is present on the network, or when peer to peer operation is disabled.
+
```

Comment:
I thought 'peer-to-peer' is mostly hyphenated in the spec but I may be wrong!

Reply:
It's a complete mix :). I'll sort that out.